src/forklift/cli.py

```diff
@@ -139,11 +139,14 @@ def start_lift(file_path=None):
 
     lift.process_crates_for(pallets, core.update)
 
+    lift.process_pallets(pallets)
+
```

Comment:
it might be nice to see the elapsed time of processing and data copying. What do you think?